tests/stacks/aws/test_data/expected/rbac.authorization.k8s.io_v1_role_centraldashboard.yaml

```diff
@@ -24,5 +24,6 @@ rules:
   - ""
   resources:
   - secrets
+  - configmaps
```

Comment:
Is this change intended?

Reply:
Right, I did a search, should be this PR: https://github.com/kubeflow/manifests/pull/1394.

The PR's `run test` actually failed after merged. Developers may forget to re-generate tests before merge. Thus, it falls back to my side when I generate tests.